app/styles/_integrations.less

```diff
@@ -0,0 +1,43 @@
+.service-integration {
+  width: 33%;
+  .image-icon {
+    font-size: 45px;
+    line-height: 1.25;
+    margin: 0 15px;
+    float: left;
+    img {
+      width: 30px;
+    }
+    span {
+      opacity: .38;
+    }
+  };
+
+  .description {
```

Comment:
Think there is `pull-right` `pull-left` classes

Reply:
nice one, updated.